homeassistant/components/screenlogic/binary_sensor.py

```diff
@@ -19,16 +19,16 @@
 async def async_setup_entry(hass, config_entry, async_add_entities):
     """Set up entry."""
     entities = []
-    data = hass.data[DOMAIN][config_entry.entry_id]
-    coordinator = data["coordinator"]
+    coordinator = hass.data[DOMAIN][config_entry.entry_id]["coordinator"]
+
+    # Generic binary sensor
+    entities.append(ScreenLogicBinarySensor(coordinator, "chem_alarm"))
 
-    for binary_sensor in data["devices"]["binary_sensor"]:
-        entities.append(ScreenLogicBinarySensor(coordinator, binary_sensor))
     async_add_entities(entities)
```

Comment:
```suggestion
    async_add_entities([ScreenLogicBinarySensor(coordinator, "chem_alarm")])
```

Reply:
I was keeping this setup because I'm going to add additional binary_sensor entities with the next update.